### ISSUE TITLE
Manifest order warning: uses-permission tag appears after application tag

### ISSUE THREAD

**Issue by XieEDeHeiShou** [checkstyle/checkstyle#7079]
I'm using Checkstyle plugin 5.32.0 on Android Studio, and I'm using built-in google style as the configuration.  
After reformat the Manifest.xml by <kbd>Ctrl + Alt + L</kbd>, the `uses-permission` tag is moved after `application` tag and a warning is shown.
![TIM截图20190920113754](https://user-images.githubusercontent.com/16363180/65298095-c6e83a80-db9c-11e9-8443-455965e99772.png)

```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
  package="demo">

  <application
    android:icon="@mipmap/ic_launcher"
    android:label="@string/app_name"
    android:theme="@style/AppTheme">
    <activity android:name=".MainActivity">
      <intent-filter>
        <action android:name="android.intent.action.MAIN" />

        <category android:name="android.intent.category.LAUNCHER" />
      </intent-filter>
    </activity>
  </application>
  <!--  <uses-permission> appears after <application> tag  -->
  <uses-permission android:name="android.permission.WRITE_CONTACTS" />
  <uses-permission android:name="android.permission.READ_CONTACTS" />

</manifest>
```

---------------
Expected:

```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
  package="demo">

  <!--  <uses-permission> expected to appears before <application> tag  -->
  <uses-permission android:name="android.permission.WRITE_CONTACTS" />
  <uses-permission android:name="android.permission.READ_CONTACTS" />

  <application
    android:icon="@mipmap/ic_launcher"
    android:label="@string/app_name"
    android:theme="@style/AppTheme">
    <activity android:name=".MainActivity">
      <intent-filter>
        <action android:name="android.intent.action.MAIN" />

        <category android:name="android.intent.category.LAUNCHER" />
      </intent-filter>
    </activity>
  </application>
</manifest>
```

--------------

**Comment by romani** [comment]
Please report issue to Android studio plugin

**Comment by XieEDeHeiShou** [comment]
Ok
